apps/src/templates/projects/ProjectsGallery.jsx

```diff
@@ -57,56 +77,43 @@ class ProjectsGallery extends Component {
     selectGallery: PropTypes.func.isRequired
   };
 
-  constructor(props) {
-    super(props);
-
-    this.toggleToGallery = this.toggleToGallery.bind(this);
-    this.toggleToMyProjects = this.toggleToMyProjects.bind(this);
-  }
-
-  toggleToGallery() {
-    window.history.pushState(null, null, '/projects/public');
-    this.props.selectGallery(Galleries.PUBLIC);
+  componentWillMount() {
+    // Add "My Libraries" tab if user is in experiment.
+    // TODO: Add myLibrariesTab to galleryTabs once experiment is complete.
```

Comment:
Is this tracked in a JIRA ticket?

Reply:
good call -- there is now! [STAR-1001](https://codedotorg.atlassian.net/browse/STAR-1001)